src/fstests/README.md

```diff
@@ -0,0 +1,148 @@
+KernelCI fstests PoC
+====================
+
+
+## Overview
+
+This page contains information about the Proof-of-Concept (PoC) project to run
+fstests automatically using KernelCI's new API.  It is based on `kvm-xfstests`
+which starts a local VM with `kvm` natively on the host where it is running.
+This solution does not involve any Docker container as all the environment is
+included in the VM image.
+
+
+## Prerequisites
+
+A number of dependencies need to be installed in oder to run `kvm-xfstests` and
+also to build a Linux kernel natively.
+
+For running `kvm-xfstests`:
+
+    sudo apt install -y qemu-kvm
+
+For building `xfstests`: see the documentation about how to [build without a
+chroot](https://github.com/tytso/xfstests-bld/blob/master/Documentation/building-xfstests.md#without-a-build-chroot).  Typically:
+
+    sudo apt install -y \
+        autoconf autoconf2.64 automake autopoint bison build-essential \
+        ca-certificates debootstrap e2fslibs-dev ed fakechroot gettext git \
+        golang-1.13-go libblkid-dev libdbus-1-3 libgdbm-dev libicu-dev \
```

Comment:
golang version may be different..are we keeping 1.15 for any special reason? I noticed as well that the version in the xfstests script is 1.11

Reply:
I changed it to `1.13` as that's what worked when I installed it.  It's not guaranteed to work on every Debian distro, some adjustments may need to be made.  This is just an indication of the dependencies.